config.json

```diff
@@ -308,7 +307,7 @@
       "topics": [
         "match",
         "result",
-        "str_vs_string",
```

Comment:
I think we should not remove the `str_vs_string` topic from any exercises. While that's not in the canonical topics list, it's an important distinction that is unique to Rust; it's useful to retain even if we also add the `strings` topic.